Dev/Kubernetes/minikube.5s.sh

```diff
@@ -13,7 +13,6 @@ MINIKUBE_LOGO_BW='iVBORw0KGgoAAAANSUhEUgAAAIAAAAB8CAYAAAChbripAAAABGdBTUEAALGPC/
 
 LANG="en_US.UTF-8"
 # You may need to update the PATH to include minikube and kubectl
-#PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
This line only prepends `/usr/local/sbin` to the default path but `minikube` or `kubectl` won't   be there, so this line essentially does nothing.

Reply:
For reference, this is the original commit for `meta_package_manager.7h.py`: https://github.com/kdeldycke/meta-package-manager/commit/f35abda586d4275f730c5a9e74805a8a1eb6d35a